tarantool_test.go

```diff
@@ -691,6 +692,59 @@ func TestClient(t *testing.T) {
 	if val != 11 {
 		t.Errorf("5 + 6 == 11, but got %v", val)
 	}
+
+	// Check for skip SQL tests if tarantool version < 2.0.0
+	resp, err = conn.Eval("return box.info.version", []interface{}{})
+	if err != nil {
+		t.Errorf("Failed to Eval for checking Tarantool version")
+	}
+	ver := parseTarantoolVersion(resp.Data[0].(string))
+	if !sqlSupported(ver) {
+		return
+	}
+
+	// SQL Execute
+	// prepare map for sql bind
+	sqlBind := map[string]interface{}{
+		"id":   1,
+		"name": "test",
+	}
+	// insert data using sql query
+	resp, err = conn.Execute("INSERT INTO SQL_TEST VALUES (:id, :name);", sqlBind)
+	if err != nil {
+		t.Errorf("Failed to Execute: %s", err.Error())
+	}
+	if resp == nil {
+		t.Errorf("Response is nil after Execute")
+	}
+	if resp.Code != 0 {
+		t.Errorf("Failed to Execute: %d", resp.Code)
+	}
+
+	sqlSelectBind := map[string]interface{}{"name": "test"}
+	// select data using sql query
+	resp, err = conn.Execute("SELECT id, name FROM SQL_TEST WHERE name=:name", sqlSelectBind)
+	if err != nil {
+		t.Errorf("Failed to Execute: %s", err.Error())
+	}
+	if resp == nil {
+		t.Errorf("Response is nil after Execute")
+	}
+	if resp.Data[0] == sqlBind["id"] && resp.Data[1] == sqlBind["name"] {
+		t.Errorf("Select failed")
+	}
+}
+
+// for internal use
+func parseTarantoolVersion(ver string) string {
+	i := strings.Index(ver, "-")
+	ver = ver[:i]
+	return ver
+}
+
+func sqlSupported(ver string) bool {
+	num, _ := strconv.Atoi(string(ver[0]))
```

Comment:
We have a helper for version comparison: https://github.com/tarantool/go-tarantool/blob/bec9f72b8e4fc7aadf02a3bced221639158fee2a/test_helpers/main.go#L103